deploy/compose/.env

```diff
@@ -1,4 +1,5 @@
 TRUST_IMAGE=ghcr.io/trustification/trust
+TRUST_VERSION=0.1.0-nightly.25a7c2ea
```

Comment:
Is this intentional? It kinda obviates the "mandatory" syntax in the compose files. Personally, I'd rather this be a "moving" tag like `latest` or unset, requiring the user to specify. Otherwise, it's gonna become old quickly.

Reply:
It was, in order be able to start it. Not having set it breaks it.

`latest` might be a reasonable default. I used nightly to ensure that it will will start up tomorrow the same way.